### DISCUSSION TITLE
Reduce motion option

### DISCUSSION THREAD

**Discussion by SK-Fast** [ppy/osu#16100] [Ideas]
Sometimes and someone gets seizure from using lazer(including me lol) from lots of effects in the game. So there should be option to reduce motion, It will reduce some animations from default to fade in/fade out

**Comment by peppy** [comment]
Maybe you're trying to be funny, but seizures are not caused by motion.

Please specify which motion is of issue to you. Although I may suggest not playing games if motion is a concern to your health.
